doc/5_saving.md

```diff
@@ -471,6 +473,53 @@ val table = TableDef("test","words",Seq(p1Col),Seq(c1Col, c2Col),Seq(rCol))
 val rddOut = rdd.map(s => outData(s._1, s._2(0), s._2(1), s._3))
 rddOut.saveAsCassandraTableEx(table, SomeColumns("col1", "col2", "col3", "col4"))
 ```
+## Deleting rows and columns
+`RDD.deleteFromCassandra(keyspaceName, tableName)` delete rows from specified Cassandra table, using data from the RDD as primary keys.
```

Comment:
I think the wording here is a little confusing. Maybe more like?

deletes row from the specified Cassandra table. The values in the RDD are interpreted as Primary Key Constraints to remove.